packages/components/organisms/f-footer/src/tenants/en-GB.js

```diff
@@ -242,6 +242,13 @@ export default {
                     rel: 'noopener',
                     gtm: 'click_about_modern_slavery_statement'
                 },
+                {
+                    url: '/CmsAssets/media/Files/Common/legal/gender-pay-gap/gender-pay-gap-2019-2021',
+                    text: 'Gender Pay Gap 2019-2021',
+                    target: '_blank',
```

Comment:
Could we use a named target instead of `_blank` so that multiple clicks doesn't open more than one window?

Reply:
I think as this is copying the current link format in the footer, this should be raised as a separate ticket so we can make that change as an Accessibility Excellence Ticket.

@simonsandersje @BanksySan Can you catch up to create this ticket at some point (if you both think it's worth doing)